en, zh: update aws deploy doc (#902)
pingcap/docs-tidb-operator: en/deploy-on-aws-eks.md

kind: ClusterConfig
metadata:
  name: ${clusterName}
  region: ap-northeast-1

nodeGroups:
  - name: admin
    desiredCapacity: 1
    privateNetworking: true
    labels:
      dedicated: admin

>   - name: tidb-1a

Reviewer: TiDB doesn't have to be deployed 3 replicas.

Author: Updated

### Scale out EKS node group

The following example shows how to scale out the `tikv-1a` group of the `${clusterName}` cluster to 2 nodes:

{{< copyable "shell-regular" >}}

```shell

> eksctl scale nodegroup --cluster ${clusterName} --name tikv-1a --nodes 2 --nodes-min 2 --nodes-max 2

Reviewer: Scaling out needs to be evenly in 3 AZs for TiKV.

Author: Updated.

### Scale out EKS node group

> For TiKV, we need to scale out evenly among the different availability zones, and the following example shows how to scale out the `tikv-1a`, `tikv-1c`, and `tikv-1d` groups of the `${clusterName}` cluster to 2 nodes:

Reviewer 2: ```suggestion
When scaling out TiKV, the node groups must be scaled out evenly among the different availability zones. The following example shows how to scale out the `tikv-1a`, `tikv-1c`, and `tikv-1d` groups of the `${clusterName}` cluster to 2 nodes:
```
